### PR TITLE
add spanish language - [WEB-5403]

### PR DESCRIPTION
**Pull request by StefonSimmons**
### What does this PR do? What is the motivation?

publishes Spanish `es` pages. Generally, follows the **How to Add another langauge to Docs** how to [here](https://datadoghq.atlassian.net/wiki/spaces/WEB/pages/3213000848/How+to+add+another+language+to+docs).
  - fixes markdown (`.md`) errors that prevented the `preview` from building.
  - fixes scss local build issues by adding the `--poll 5s` option to `hugo server` cmd. [more detail on issue](https://github.com/gohugoio/hugo/issues/12649)
       ```bash
       ERROR TOCSS: failed to transform "styles/style.scss" (text/x-scss)...File to import not found or unreadable
       ```
      - considering upgrading to `hugo` [v0.134.0](https://github.com/gohugoio/hugo/releases/tag/v0.134.0) for the `golibsass` minor version update 

**SOFT LAUNCH**:
1. typsense search wont work for spanish pages until the typesense nightly build.
    - I removed **Espanol** from the language dropdown via some conditions in the [language-region-select.html partial](https://github.com/DataDog/documentation/pull/25922/files#diff-758902bbdd08727957f4667c1eb20fa75c1f1bd0db6858a60707409ab470be54)
    - will remove conditions once search works on spanish pages. 
    - will include `es` in the lang-redirect.js script `allowedLanguages` and `redirectLanguages` arrays

**motivation**:  https://datadoghq.atlassian.net/jira/software/projects/WEB/boards/102?selectedIssue=WEB-5403
**preview**: https://docs-staging.datadoghq.com/stefon.simmons/enable-es-lang-web-5403/
 - Spanish Site: https://docs-staging.datadoghq.com/stefon.simmons/enable-es-lang-web-5403/es/

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: .gitlab-ci.yml
Comment (StefonSimmons) [inline .gitlab-ci.yml:1]: fixes a CI build issue https://gitlab.ddbuild.io/DataDog/documentation/-/jobs/690016726#L1969

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -211,7 +211,7 @@ build_preview:
       - typesense.config.json
   after_script:
     - |
-      if [ -s $PREVIEW_ERROR_LOG && $CI_JOB_STATUS != 'success' ]; then
+      if [ -s $PREVIEW_ERROR_LOG ] && [ $CI_JOB_STATUS != 'success' ]; then
```
Comment (StefonSimmons) [inline .gitlab-ci.yml:214]: fixes a CI build issue https://gitlab.ddbuild.io/DataDog/documentation/-/jobs/690016726#L1969

File: assets/scripts/tests/lang-redirects.test.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ describe(`Ensure EN site loads and redirects as expected `, () => {
     });
 
     it('should ignore when lang_pref is not in allowedLanguages', () => {
-        window.location.href = 'http://localhost:3000/?lang_pref=es';
+        window.location.href = 'http://localhost:3000/?lang_pref=it';
```
Comment (StefonSimmons) [inline assets/scripts/tests/lang-redirects.test.js:66]: fixes test now that `es` is included in the `allowedLanugages`

File: config/_default/params.es.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+site_name: "Datadog Infrastructure and Application Monitoring"
+meta_title: Getting Started with Datadog
+meta_description: Datadog, the leading service for cloud-scale monitoring.
+disclaimer: "This page is not yet available in English, we are working on its translation. <br> May you have any questions or feedback about our current translation project, <a href=\"https://docs.datadoghq.com/fr/help/\">feel free to reach out to us!</a>"
```
Comment (rtrieu) [inline config/_default/params.es.yaml:4]:
```suggestion
disclaimer: "This page is not yet available in English, we are working on its translation. <br> Should you have any questions or feedback about our current translation project, <a href=\"https://docs.datadoghq.com/fr/help/\">feel free to reach out to us!</a>"
```

Comment (StefonSimmons) [inline config/_default/params.es.yaml:4]:
```suggestion
disclaimer: "This page is not yet available in Spanish, we are working on its translation. <br> May you have any questions or feedback about our current translation project, <a href=\"https://docs.datadoghq.com/fr/help/\">feel free to reach out to us!</a>"
```

Comment (kayayarai) [reply config/_default/params.es.yaml:4]:
```suggestion
disclaimer: "This page is not yet available in Spanish, we are working on its translation. <br> If you have questions or feedback about our current translation project, <a href=\"https://docs.datadoghq.com/es/help/\">feel free to reach out to us!</a>"
```

**Review by kayayarai — APPROVED** [review]

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```
Comment (StefonSimmons) [inline local/bin/py/build/configurations/pull_config_preview.yaml:3]:
```suggestion
    cache_enabled: true
```

**Review by LisianeTu — APPROVED** [review]

**Review by bgdeutsch — APPROVED** [review]
LGTM 👍

File: config/_default/params.es.yaml (hunk 3 above)
Comment (rtrieu) [reply config/_default/params.es.yaml]: One minor tweak.

**Review by rtrieu — APPROVED** [review]

**Review by aliciascott — APPROVED** [review]

**Pull request merged**